docs/modules/general/proto.mdx

```diff
@@ -42,10 +42,40 @@ Mapmakers should always use the latest supported proto version, and this may be
 ### Changes in 1.5.0
 
 :::note
-`1.5.0` is a upcoming protocol version that is currently in its planning stage and therefore, **not implemented**.
-If you would like to learn more or propose changes, please visit [PGM#1267 on the issue tracker](https://github.com/PGMDev/PGM/issues/1267).
+`1.5.0` is now available. As the latest protocol version, it is not fully polished, so bugs are expected.
+If you would like to learn more or report issues, please visit the [PGM issue tracker](https://github.com/PGMDev/PGM/issues).
 :::
 
+#### Breaking
+
+- Filters and variables are now their own IDs that will be reserved by PGM. Those IDs can be referenced anywhere in the XML.
```

Comment:
This wording makes it sound like all filters/variables behave like this; that is not the case.
This is only true for some filters & variables (the ones that essentially took no extra parameters), note that this isn't a new behavior, things like `never` or `everywhere` did exactly the same in the past, there's just now a bunch more reserved ids for commonly-used filters.

Reply:
I completely forgot about `never` and `everywhere`. Were there any variables with this behavior before proto 1.5.0? If not, would this be a good change?

```suggestion
- Variables and some additional filters are now their own IDs that will be reserved by PGM. Those IDs can be referenced anywhere in the XML.
```

Reply:
Variables were all added at modern times, the latest proto (1.4.2) pre-dates them by years, so no we could not have reserved any id for them.
The updated text looks better, but it still sounds like it applies to all variables, maybe `Some variables and some additional filters`?